tests/msbuild/Xamarin.MacDev.Tests/ProjectsTests/CodesignAppBundle.cs

```diff
@@ -26,12 +26,13 @@ public CodesignAppBundle (string platform, string configuration)
 		static bool IsCodesigned (string path)
 		{
 			var psi = new ProcessStartInfo ("/usr/bin/codesign");
-			var args = new CommandLineArgumentBuilder ();
+			var args = new List<string> ();
 
 			args.Add ("--verify");
-			args.AddQuoted (path);
+			args.Add (path);
```

Comment:
Are we missing something by not doing the same as `AddQuoted`?

Reply:
No, because arguments are automatically quoted later: https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.processstartinfo.argumentlist?view=net-9.0.